packages/main/src/plugin/color-registry.ts

```diff
@@ -411,6 +412,64 @@ export class ColorRegistry {
     });
   }
 
+  protected initContent(): void {
+    const ct = 'content-';
+    this.registerColor(`${ct}breadcrumb`, {
+      dark: colorPalette.gray[600],
+      light: colorPalette.purple[900],
+    });
+
+    this.registerColor(`${ct}breadcrumb-2`, {
+      dark: colorPalette.purple[400],
+      light: colorPalette.purple[600],
+    });
+
+    this.registerColor(`${ct}header`, {
+      dark: colorPalette.white,
+      light: colorPalette.charcoal[900],
+    });
+
+    this.registerColor(`${ct}header-icon`, {
+      dark: colorPalette.gray[600],
+      light: colorPalette.gray[300],
+    });
+
+    this.registerColor(`${ct}card-header-text`, {
+      dark: colorPalette.gray[100],
+      light: colorPalette.charcoal[900],
+    });
+
+    this.registerColor(`${ct}card-bg`, {
+      dark: colorPalette.charcoal[800],
+      light: colorPalette.gray[50],
+    });
+
+    this.registerColor(`${ct}card-text`, {
+      dark: colorPalette.gray[400],
+      light: colorPalette.purple[900],
+    });
+
+    this.registerColor(`${ct}card-inset-bg`, {
+      dark: colorPalette.charcoal[900],
+      light: colorPalette.dustypurple[200],
+    });
+
+    this.registerColor(`${ct}bg`, {
+      dark: colorPalette.charcoal[700],
+      light: colorPalette.gray[300],
+    });
+
+    this.registerColor(`${ct}card-icon`, {
+      dark: colorPalette.gray[400],
+      light: colorPalette.purple[900],
+    });
+
+    this.registerColor(`${ct}divider`, {
+      dark: colorPalette.charcoal[400],
+      light: colorPalette.gray[700],
+    });
```

Comment:
This one is not defined in the PDF referenced in the PR description, but is discussed here: https://github.com/containers/podman-desktop/issues/6929#issuecomment-2097884396

The name `divider` is a proposal, open to discussion